app/views/pieces/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<div id="chess">
+  <h1>Jedi Chess</h1>
+    <table>
+      <!-- create a 8x8 table using ruby loops, for each td tag, also insert the id with the position value such as a8, b6, c3..etc. The id will be used to insert the pieces later -->
+      <% y_array = (1..8).to_a.reverse! %>
+      <% y_array.each do |y_value| %>
+        <%= content_tag "tr" do %>
+          <% x_array = ('a'..'h').to_a %>
+          <% x_array.each do |x_value| %>
+            <% pos_id = x_value + y_value.to_s %>
+            <%= content_tag "td", id: pos_id do %>
+            <% end %>
+          <% end %>
+        <% end %>
+      <% end %>
+    </table>
+</div>
+
+
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+<script>
+  var pieces_hash = <%= raw @pieces_hash.to_json %>; //first convert a ruby hash to json format so js can utilize using .to_json
+	
+	var current_id = '#' + <%= raw @piece_pos_id.inspect %> //not sure why I need inspect and raw here
```

Comment:
Without the inspect method, the pieces are not rendered on the chess board. Weird